js/interface.js

```diff
@@ -1193,6 +1193,38 @@ var DynamicLists = (function() {
       _this.handleTokensSelection();
       _this.loadTokenFields();
     },
+    sortDataSourcesListByCurrentAppId: function(dataSources) {
+      var groupedDataSources = [
+        {
+          id: 'currentAppDSs',
+          text: 'This app',
+          name: 'currentApp',
+          children: []
```

Comment:
In what order are the data sources listed within each group?

Reply:
@tonytlwu it is not sorted at all. I will add sorting as you have described.